dftool

```diff
@@ -4,6 +4,7 @@ import datetime
 import functools
 import os
 import sys
+from calendar import WEDNESDAY
```

Comment:
@calina-c you can remove the changes in dftool. I will handle them as part of [issue#540](https://github.com/oceanprotocol/df-py/issues/540) // [PR#554](https://github.com/oceanprotocol/df-py/pull/554).